### DISCUSSION TITLE
Q-Form and rules validation in v-for

### DISCUSSION THREAD

**Discussion by micst96** [quasarframework/quasar#17294] [General - Components / Directives / etc]
Hi, I have problem with q-input validation for v-for div. Inside v-for I have many q-inputs and rules are validated only when I manually click on field, but when I click on button to go to the next page, greedy validation does not work and allow to proceed to the next page, while q-input inside v-for div has empty value that is not allowed.

`<q-form greedy @submit="nextStep()">` - this is q-form
`<q-btn type="submit" color="cyan-7" no-caps outline dense class="q-px-md" label="Next"/>` - submit button
inside q-form I have q-input on top level that is working and rules prevent to go to next step, q-input inside v-for does not work

**Comment by stefanvanherwijnen** [comment]
You need to call `myFormRef.value.validate()` in 'nextStep'.

https://quasar.dev/vue-components/form/
